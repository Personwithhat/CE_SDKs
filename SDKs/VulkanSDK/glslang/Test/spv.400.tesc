version https://git-lfs.github.com/spec/v1
oid sha256:7332dbdcbab42a1778e4dc4958c4448bcd25b32ca3c140aac761030a68f2d614
size 1055
