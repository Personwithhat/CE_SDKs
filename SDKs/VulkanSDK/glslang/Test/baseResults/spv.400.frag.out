version https://git-lfs.github.com/spec/v1
oid sha256:dee8f59887a6eb19a63cbb3dca4f7b1bd66ff5d70335156f87d0ee976ce4187a
size 70356
